tests/search.rs

```diff
@@ -84,9 +84,7 @@ fn simple() {
     assert_that(cargo_process("search").arg("postgres")
                     .arg("--host").arg(registry().to_string()),
                 execs().with_status(0)
-                       .with_stderr("\
-[UPDATING] registry `[..]`")
```

Comment:
Ah I just changed the clauses later to `with_stderr_contains` to avoid matching this output, which we in theory didn't really need to match anyway.